src/components/FlowSelector.js

```diff
@@ -34,7 +31,7 @@ class FlowSelector extends React.Component {
       to = "/eligibility/cities/" + town;
     } else if (county in regulatedCounties) {
       to = "/eligibility/counties/" + county;
-    } else if (town != "") {
+    } else if (town !== "") {
```

Comment:
Good catch on this bug fixes, but can we try and put these in separate diffs because it makes it much easier to review.